lib/router/router.dart

```diff
@@ -31,6 +37,11 @@ import '../screens/search/explore_category_page.dart';
     MaterialRoute<void>(page: ContactSupport, path: 'contact-support'),
     MaterialRoute<void>(page: AboutScreen, path: 'about'),
     MaterialRoute<void>(page: FAQ, path: 'faq'),
+    CustomRoute<void>(
```

Comment:
does this need to be here?
If yes, use the variable you created.
If no, you can remove it :)

Reply:
It's not needed. Forgot to remove it.